src/index.html

```diff
@@ -7,21 +7,396 @@
+            <p class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <div class="product-code">Product code: 195434</div>
+            <div class="product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product-rating__reviews">Reviews: 5</p>
+            </div>
+            <div class="product-price product-price--container">
+              <p class="product-price__text">Price:</p>
+              <p class="product-price__value">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="buy-button"
+              role="button"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <div class="product-card__image"></div>
+          <div class="product-card__content">
+            <p class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <div class="product-code">Product code: 195434</div>
+            <div class="product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product-rating__reviews">Reviews: 5</p>
+            </div>
+            <div class="product-price product-price--container">
+              <p class="product-price__text">Price:</p>
+              <p class="product-price__value">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="buy-button"
+              role="button"
+              data-qa="hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <div class="product-card__image"></div>
+          <div class="product-card__content">
+            <p class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <div class="product-code">Product code: 195434</div>
+            <div class="product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
```

Comment:
Same as previous comment, the stars rating system should be implemented with CSS classes.